lib/models/food.js

```diff
@@ -0,0 +1,33 @@
+const environment = process.env.NODE_ENV || 'development';
+const configuration = require('../../knexfile')[environment];
+const database = require('knex')(configuration);
+
+function createFoods(name, calories) {
+  return database.raw(`INSERT INTO foods (name, calories, created_at) VALUES (?, ?, ?)`,
+  [name, calories, new Date])
+}
+
+function resetFoods() {
+  return database.raw(`TRUNCATE foods RESTART IDENTITY`)
+}
+
+function findFirst() {
+  return database.raw('SELECT * FROM foods LIMIT 1')
+}
+
+function find(id) {
+  return database.raw("SELECT * from foods where id=?", [id])
+}
+
+function findAll() {
+  return database.raw("SELECT * FROM foods;")
+}
+
```

Comment:
Great instincts for abstracting this logic. It's almost like making your own ActiveRecord!